public/mobile-app/src/lib/state/User.svelte.test.ts

```diff
@@ -90,6 +90,7 @@ describe('/lib/state/User.svelte.ts', () => {
 
     // Then
     expect(userStore.connected?.identity?.address).toEqual(mockUserIdentity.address)
+    expect(userStore.connected?.identity?.address instanceof Address).toBe(true)
```

Comment:
Is this needed? I mean, shouldn't typescript warn us about assigning anything other than an `Address` to `userStore.connected.identity.address`?

If not, I'm curious as to why, and maybe we should investigate (because one of the benefits of using typescript is to have this kind of protection, and thus less need for that kind of unit tests).